### PR TITLE
Rename httputils package to utils

### PR DESCRIPTION
**Pull request by tock-ibm**
Rename `httputils` package to `utils`
Rename file `utils.go` to `http.go`

No functional change.

### CODE REVIEW

File: internal/utils/blocks_test.go [hunk 1]
```diff
@@ -1,13 +1,13 @@
 // Copyright IBM Corp. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package httputils_test
+package utils_test
```
Comment (gennadylaventman) [inline internal/utils/blocks_test.go:4]: Why do we need new package here?

Comment (gennadylaventman) [reply internal/utils/blocks_test.go:4]: Not related to this pr exactly, but you change package name in any case

Comment (tock-ibm) [reply internal/utils/blocks_test.go:4]: I like black box testing to check exported methods. 
Anyway, I changed it.

Comment (gennadylaventman) [reply internal/utils/blocks_test.go:4]: ok

**Review by gennadylaventman — APPROVED** [review]

**Pull request merged**